x-pack/plugins/security_solution/server/endpoint/services/actions/create/types.ts

```diff
@@ -22,6 +22,7 @@ export type CreateActionPayload = TypeOf<typeof ResponseActionBodySchema> & {
   rule_name?: string;
   error?: string;
   hosts?: Record<string, { name: string }>;
+  action_id?: string;
```

Comment:
I'm assuming this is absolutely needed?
I'll have to re-evaluate this once you merge and I pull this back into my [refactor PR](https://github.com/elastic/kibana/pull/174691)

Reply:
I am using this to include the action_id in case attachment for reference. I can keep it outside of createPayload, but this seemed to be a good place.

Reply:
I removed it, not going to use the createPayload in the end. Thanks for the hint!